.travis.yml

```diff
@@ -15,6 +15,7 @@ env:
   - BASE_DISTRO: ubuntu:precise
   - BASE_DISTRO: ubuntu:trusty
   - BASE_DISTRO: ubuntu:xenial
+  - BASE_DISTRO: ubuntu:beaver
```

Comment:
Shouldn't this be 'bionic' ?

Reply:
Comment correct, but not from this pr. It just needs to be correctly rebased over develop